src/meta/src/stream/stream_manager.rs

```diff
@@ -623,9 +612,32 @@ where
     /// Dropping materialized view is done by barrier manager. Check
     /// [`Command::DropMaterializedView`] for details.
     pub async fn drop_materialized_view(&self, table_id: &TableId) -> Result<()> {
+        let table_fragments = self
+            .fragment_manager
+            .select_table_fragments_by_table_id(table_id)
+            .await?;
+
+        let mut source_fragments = HashMap::new();
+        fetch_source_fragments(&mut source_fragments, &table_fragments);
+
         self.barrier_manager
             .run_command(Command::DropMaterializedView(*table_id))
             .await?;
+
+        let mut actor_ids = HashSet::new();
```

Comment:
Ditto.

Reply:
Using Default::default here would make it impossible to derive the type

Reply:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/1997790/173385682-13687c49-8b33-47be-bd1a-73cf83ce8db1.png">